pkg/controller/jobframework/reconciler.go

```diff
@@ -451,6 +455,28 @@ func (r *JobReconciler) ReconcileGenericJob(ctx context.Context, req ctrl.Reques
 	return ctrl.Result{}, nil
 }
 
+func (r *JobReconciler) recordAdmissionCheckUpdate(wl *kueue.Workload, job GenericJob) {
+	message := ""
+	object := job.Object()
+	for _, check := range wl.Status.AdmissionChecks {
+		if check.State == kueue.CheckStatePending && check.Message != "" {
+			if message != "" {
+				message += "; "
+			}
+			message += "Admission check: \"" + check.Name + "\", Message: \"" + check.Message + "\""
```

Comment:
That's a lot of extra characters.
Maybe just `check.Name: message`

Reply:
Done.